typescript/packages/common-runner/src/builtins/llm.ts

```diff
@@ -61,14 +61,44 @@ export function llm(
     const { system, messages, prompt, stop, max_tokens, model } =
       inputsCell.getAsQueryResult([], log) ?? {};
 
-    const hash = refer({
+    // Define types for our parameters
+    type BaseParams = {
+      model: string;
+      messages: SimpleContent[] | SimpleMessage[];
+      max_tokens: number;
+    };
+
+    type StandardParams = BaseParams & {
+      system: string;
+      prompt: string;
+      stop: string;
+    };
+
+    type O1Params = BaseParams;
+
+    let llmParams: StandardParams | O1Params = {
       system: system ?? "",
       messages: messages ?? [],
       prompt: prompt ?? "",
       stop: stop ?? "",
       max_tokens: max_tokens ?? 4096,
-      model: model ?? "claude-3-5-sonnet-latest",
-    }).toString();
+      model: model ?? "openai:o1-preview",
+    } as StandardParams;
+
+    if (model?.startsWith("openai:o1")) {
```

Comment:
I don't think it will be just o1 ... many vision models don't support system messages